docs/how-to/policy-guard/policy_guard.mdx

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 1
+title: Policy Guard
+---
+
+Policy Guards provide centralized control over permissions and roles across projects within an organization. This allows for consistent, organization-wide enforcement of rules regarding access and actions within projects.
+
+---
+
+## What is a Policy Guard?
+
+A Policy Guard defines specific permissions for roles across multiple projects.
+For example, if you need only "admin" roles to create documents, while "viewer" roles have read-only access, you can enforce these rules through a Policy Guard, ensuring that all projects included in this policy adhere to the same permissions consistently.
+
+## Key Capabilities of Policy Guards
+
+- **Centralized Control:** Manage permissions and roles at the organization level, allowing for streamlined rule-setting across projects.
+- **Flexible Role Management:** Define what each role can or cannot do across projects included in the policy.
+- **Project Inclusion and Exclusion:** Easily add or remove projects from a policy guard to control their permissions.
+- **Permission Rules:** Set specific permission rules, such as granting or revoking permissions for certain roles to perform actions on resources (e.g., creating, reading documents).
+
+## Example Use Cases
+- **Role-Based Access:** Assign specific actions to roles (e.g., granting "admin" the ability to create documents and limiting "viewer" to read-only).
+- **Multi-Project Enforcement:** Enforce the same permissions across multiple projects, ensuring consistent security standards.
+- **Audit and Compliance:** Review and list current permissions applied across various projects, allowing for easy auditing and compliance checks.
```

Comment:
In the example, lets first give the goal - we want to enforce that an admin will be able to read a document across all projects, and that no other role will be able to read document. 
In addition, It's important to add that only an "Workspace Owner" will be able to edit these Policy Guard Rules.